library/spdm_responder_lib/libspdm_rsp_receive_send.c

```diff
@@ -12,6 +12,89 @@ typedef struct {
     libspdm_get_spdm_response_func get_response_func;
 } libspdm_get_response_struct_t;
 
+#if LIBSPDM_DEBUG_PRINT_ENABLE
```

Comment:
This piece of code is generic for both requester and response.
Why not put to common lib?

Reply:
correct. i was focusing on the responder side. i will make the change to be applicable for both